Grading at 12/03/2022:21:58:43.089606
For commit ID 716b2c042c5e1d4d2c23c883d5ff6df69b3e1af5
Grading at Sat Dec  3 04:58:35 PM EST 2022
Doing some basic checking for your submission

Step 1
Checking that cyoa-step1 compiles
g++ -o cyoa-step1 -std=gnu++98 -Werror -Wall -pedantic -ggdb3 cyoa-step1.cpp
The code compiled (good)
Executable cyoa-step1 exists (good)
------------------------------------
Now running YOUR tests for step 1
------------------------------------


1.1: Current test is ./cyoa-step1 numfault1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
terminate called after throwing an instance of 'InvalidInputLine'
  what():  Invalid input line!

timeout: the monitored command dumped core
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.2: Current test is ./cyoa-step1 numfault2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
terminate called after throwing an instance of 'InvalidInputLine'
  what():  Invalid input line!

timeout: the monitored command dumped core
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.3: Current test is ./cyoa-step1 numfault3 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
terminate called after throwing an instance of 'NumOutOfRange'
  what():  Number out of range!

timeout: the monitored command dumped core
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.4: Current test is ./cyoa-step1 goodstory < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
terminate called after throwing an instance of 'InvalidChoiceAdd'
  what():  Can't add choice to a WIN/LOSS page!

timeout: the monitored command dumped core
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 1

Step 2
Checking that cyoa-step2 compiles
g++ -o cyoa-step2 -std=gnu++98 -Werror -Wall -pedantic -ggdb3 cyoa-step2.cpp	
The code compiled (good)
Executable cyoa-step2 exists (good)
------------------------------------
Now running YOUR tests for step 2
------------------------------------
No request to check for intermediate deadline completion on step 2

Step 3
Checking that cyoa-step3 compiles
make: 'cyoa-step3' is up to date.
The code compiled (good)
Executable cyoa-step3 exists (good)
------------------------------------
Now running YOUR tests for step 3
------------------------------------


3.1: Current test is ./cyoa-step3 goodstory < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
terminate called after throwing an instance of 'InvalidChoiceAdd'
  what():  Can't add choice to a WIN/LOSS page!

timeout: the monitored command dumped core
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 3

Step 4
Checking that cyoa-step4 compiles
make: 'cyoa-step4' is up to date.
The code compiled (good)
Executable cyoa-step4 exists (good)
------------------------------------
Now running YOUR tests for step 4
------------------------------------
No request to check for intermediate deadline completion on step 4

You have passed the test cases YOU provided
If your tests are good, this is great. 
If your tests are insufficient, you should do much more testing

Overall Grade: NO GRADE YET
